### ISSUE TITLE
Add ability to reset XP

### ISSUE THREAD

**Issue by Mis4meow** [phpfeniks/bot#6]
Add ability to reset a user's XP or clear an entire channel of XP

**Comment by xqus** [comment]
When you say clear an entire channel do you mean XP for everybody in that channel?

**Comment by Mis4meow** [comment]
Yes, to reset the entire channel. But the ability to reset XP for channels, people, the entire server, and people who have left is necessary.

**Comment by xqus** [comment]
Partially fixed by 5b8c079b3d55e31a79480ddab2ea87f5b83de179

**Comment by xqus** [comment]
- [X] Clear individual user
- [ ] Clear the entire server
- [ ] Clean up by clearing users who have left
